From e6d347c2e475126fc906dd83efb5713501ba1a83 Mon Sep 17 00:00:00 2001
From: Bastian Blank <bastian.blank@credativ.de>
Date: Thu, 6 Mar 2014 13:37:41 +0100
Subject: Use libbsd for strlcat and strlcpy

---
 Makefile.linux | 12 +++---------
 1 file changed, 3 insertions(+), 9 deletions(-)

diff --git a/Makefile.linux b/Makefile.linux
index a37529e..4028337 100644
--- a/Makefile.linux
+++ b/Makefile.linux
@@ -7,12 +7,12 @@
 # (contributed by admin@2ka.mipt.ru)
 
 CFLAGS=		-g
-LDFLAGS=	-L/usr/lib/libmilter -lmilter -lpthread
+LDFLAGS=	-L/usr/lib/libmilter -lmilter -lpthread -lbsd
 
 all: milter-regex milter-regex.cat8
 
-milter-regex: milter-regex.o eval.o strlcat.o strlcpy.o y.tab.o
-	gcc -o milter-regex milter-regex.o eval.o strlcat.o strlcpy.o y.tab.o $(LDFLAGS)
+milter-regex: milter-regex.o eval.o y.tab.o
+	gcc -o milter-regex milter-regex.o eval.o y.tab.o $(LDFLAGS)
 
 milter-regex.o: milter-regex.c eval.h
 	gcc $(CFLAGS) -c milter-regex.c
@@ -20,12 +20,6 @@ milter-regex.o: milter-regex.c eval.h
 eval.o: eval.c eval.h
 	gcc $(CFLAGS) -c eval.c
 
-strlcat.o: strlcat.c
-	gcc $(CFLAGS) -c strlcat.c
-
-strlcpy.o: strlcpy.c
-	gcc $(CFLAGS) -c strlcpy.c
-
 y.tab.o: y.tab.c
 	gcc $(CFLAGS) -c y.tab.c
 
